scripts/vet.sh

```diff
@@ -181,6 +181,6 @@ REV_OUT="$(mktemp)"
 revive -set_exit_status=1 -exclude "reflection/test/grpc_testing_not_regenerate/" -formatter plain ./... >"${REV_OUT}" || true
 
 # TODO: Remove `|| true` to unskip linter failures once existing issues are fixed.
-(not grep -v "\.pb\.go:" "${REV_OUT}") || true
+(noret_grep -v "reflection/test/grpc_testing_not_regenerate/" "${REV_OUT}" | not grep -v "\.pb\.go:") || true
```

Comment:
i already added this [here](https://github.com/purnesh42H/grpc-go/blob/master/scripts/vet.sh#L181)

Reply:
Oh ok. Thanks. Reverted